src/react/components/common/tagInput/tagInput.tsx

```diff
@@ -611,19 +633,40 @@ export class TagInput extends React.Component<ITagInputProps, ITagInputState> {
         return menuItems;
     }
 
+    private isTypeCompatibleWithTag = (tag, type) => {
```

Comment:
add this in this method which means empty tag will be compatible with any type
if (tag && tag.documentCount <= 0) {
    return true;
}